lib/vsc/utils/fancylogger.py

```diff
@@ -443,8 +452,14 @@ def logToScreen(enable=True, handler=None, name=None, stdout=False):
 
     by default, logToScreen will log to stderr; logging to stdout instead can be done
     by setting the 'stdout' parameter to True
+
+    The `colorize` parameter enables or disables log colorization using
+    ANSI terminal escape sequences, according to the values allowed
+    in the `colorize` parameter to function `_screenLogFormatterFactory`
+    (which see).
```

Comment:
> > +    The `colorize` parameter enables or disables log colorization using
> > +    ANSI terminal escape sequences, according to the values allowed
> > +    in the `colorize` parameter to function `_screenLogFormatterFactory`
> > +    (which see).
>
> which see?

"""
The abbreviation q.v. stands for quod vide, which translates literally
as “which see,” although in practice it means something more like “for
which see elsewhere.” It is used in notes after a word or phrase to
indicate that more information can be found about the topic somewhere
else in the current work.
"""

From:
http://writingcenter.unc.edu/handouts/latin-terms-and-abbreviations/
(which see ;-))